reana-ui/src/pages/workflowList/WorkflowList.js

```diff
@@ -132,7 +132,7 @@ function Workflows() {
   );
 
   return (
-    <div className={styles.container}>
+    <div className={`${styles.container} ${styles["workflow-list-container"]}`}>
```

Comment:
I think the style `workflow-list-container` is never defined?

Reply:
Oops, my bad, I think it's a relic from an attempt to style the page differently. Removed it!